### ISSUE TITLE
Missing version in manifest causes Exception

### ISSUE THREAD

**Issue by psyonara** [OCA/pylint-odoo#231]
If the manifest file of a module does not have a version specified at all, it raises an exception and pylint doesn't continue.
```
File "/home/blah/odoo/venv/lib/python3.6/site-packages/pylint_odoo/misc.py", line 73, in formatversion
    return re.match(self.config.manifest_version_format_parsed, string)
  File "/usr/lib/python3.6/re.py", line 172, in match
    return _compile(pattern, flags).match(string)
TypeError: expected string or bytes-like object
```

I'm guessing instead of running the version check it should rather result in a warning that the version hasn't been specified?

**Comment by pedrobaeza** [comment]
IMO all manifest must have a version key.

**Comment by psyonara** [comment]
> IMO all manifest must have a version key.

I agree, so therefore the tool should have a warning/error about that, instead of breaking completely.

**Comment by pedrobaeza** [comment]
The error itself indicates the problem, but if you want to contribute with the patch, it's welcome.

**Comment by psyonara** [comment]
Version 2.0.1 doesn't fix the problem for me. I realized, though, that Pylint doesn't stop completely, but rather just prints the exception when running on that module.

**Comment by moylop260** [comment]
Could you send us the `__manifest__.py` file with the issue, please?
And the `pylint` command executed and your configuration file used (default `~/.pylintrc`)
In order to reproduce it locally.

Maybe you are using the parameter `valid_odoo_versions=INVALID VALUE`

Can you try using: `--valid_odoo_versions='.*'`?

**Comment by moylop260** [comment]
I can't reproduce it and the additional information requested was not sent it
Maybe it is a issue using python3 we have released pylint-odoo 3.0.0 supporting py3

feel free of send more information about in order to open it
